includes/definitions/mobileiron.yaml

```diff
@@ -0,0 +1,33 @@
+os: mobileiron
+type: appliance
+group: unix
+text: MobileIron
+ifXmcbc: true
+ifname: true
+mib_dir:
+    - supermicro
+    - dell
+over:
+    - { graph: device_processor, text: 'Processor Usage' }
+    - { graph: device_ucd_memory, text: 'Memory Usage' }
+    - { graph: device_storage, text: 'Storage Usage' }
+poller_modules:
+    bgp-peers: false
+    ospf: false
+    stp: false
+discovery_modules:
+    applications: true
+    bgp-peers: false
+    stp: false
+    vmware-vminfo: true
+    libvirt-vminfo: true
+processor_stacked: true
+register_mibs:
```

Comment:
why are you registering mibs for another OS?

Reply:
As it's CentOS under the hood I took the Linux definition as a start point, so I just assumed they were needed.

Looking at the definitions though, I think they shouldn't be in the Linux definition either - this seems to be the intended OS?
https://github.com/librenms/librenms/blob/master/includes/definitions/pktj.yaml

Reply:
I think they were defined in PHP, then ported to yaml.  Yeah, they could be moved specifically to pktj

Reply:
Ok, I'll submit that as a seperate request to keep things simple.